### DISCUSSION TITLE
Sensors module should use the -j (JSON) flag for more useful information

### DISCUSSION THREAD

**Discussion by darkpixel** [saltstack/salt#67619] [Feature Requests]
The current output of the sensors module is string data wrapped into a dictionary.

![Screenshot from 2021-06-29 08-56-40](https://user-images.githubusercontent.com/158380/123830657-5a918380-d8b8-11eb-862f-4acf28ba4d38.png)

The sensors application has support for the ```-j``` flag which returns data in JSON format.  Using that data returns a more structured result.

![Screenshot from 2021-06-29 08-56-49](https://user-images.githubusercontent.com/158380/123830761-772dbb80-d8b8-11eb-8950-25bcb5e7a1de.png)

A small change to the ```modules/sensors.py``` file returns the latter output:
```
    extra_args = ""
    if fahrenheit is True:
        extra_args = "-f"
    extra_args += ' -j'
    sensors = __salt__["cmd.run"](
        "/usr/bin/sensors {0} {1}".format(chip, extra_args), python_shell=False
    )
    ret = json.loads(sensors)
    return ret
```

**Comment by sagetherage** [comment]
@darkpixel thank you, you may open a Pull Request with this and we will review!

**Comment by darkpixel** [comment]
Hmm...when calling ```sensors -f``` you can get the output in fahrenheit.  Without the ```-f``` flag you get the results in Celsius....but if you call ```sensors -j``` the JSON output *always* returns Celsius.  Even if you pass the ```-f``` flag.  So this change would possibly break stuff.

**Comment by sagetherage** [comment]
ok, well, I have to put this into blocked - maybe you want to file an upstream bug?

**Comment by darkpixel** [comment]
Maybe a second module would work?  Something like ```sensorsjson```?
